Communication/Packets/Incoming/Ambassadors/SendAlertAmbassadorComposer.cs

```diff
@@ -1,23 +1,24 @@
 ﻿using Plus.HabboHotel.GameClients;
```

Comment:
It should be SendAmbassadorAlertEvent (And make sure the matching header id name has been updated). Its an event, not a composer.